share/initial-mysql.sql

```diff
@@ -18,7 +18,8 @@ CREATE TABLE test_results (
 	params_deterministic_hash character varying(32),
 	params blob NOT NULL,
 	results blob DEFAULT NULL,
-    undelegated boolean NOT NULL DEFAULT false
+    undelegated boolean NOT NULL DEFAULT false,
```

Comment:
The indentation is inconsistent here. Have you considered replacing tab characters with spaces? Personally I like to use perltidy for this purpose.